src/VirtoCommerce.WebHooksModule.Web/Module.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Modularity;
+using VirtoCommerce.Platform.Core.Security;
+using VirtoCommerce.Platform.Core.Settings;
+using VirtoCommerce.Platform.Data.Extensions;
+using VirtoCommerce.WebhooksModule.Data.Repositories;
+using VirtoCommerce.WebhooksModule.Data.Services;
+using VirtoCommerce.WebHooksModule.Core;
+using VirtoCommerce.WebHooksModule.Core.Services;
+using VirtoCommerce.WebHooksModule.Data.Services;
+
+namespace VirtoCommerce.WebHooksModule.Web
+{
+    public class Module : IModule
+    {
+        public ManifestModuleInfo ModuleInfo { get; set; }
+
+        public void Initialize(IServiceCollection serviceCollection)
+        {
+            serviceCollection.AddTransient<IWebHookRepository, WebHookRepository>();
+            serviceCollection.AddDbContext<WebhookDbContext>((provider, options) =>
+            {
+                var configuration = provider.GetRequiredService<IConfiguration>();
+                options.UseSqlServer(configuration.GetConnectionString("VirtoCommerce.Catalog") ?? configuration.GetConnectionString("VirtoCommerce"));
```

Comment:
configuration.GetConnectionString("VirtoCommerce.Catalog") ??
is it needed?

Reply:
fixed